api/tests/test_get_user.py

```diff
@@ -28,6 +28,10 @@ def test_get_user(client, table, monkeypatch):
             'limit': 25,
             'remaining': 21,
         },
+        'names': sorted([
```

Comment:
don't need `sorted` here, removing it verifies that the API is doing the sorting correctly